src/server/services/hli/stadspas-gpass-service.ts

```diff
@@ -248,23 +250,33 @@ export async function fetchGpassBudgetTransactions(
 
 function transformGpassAanbiedingenResponse(
   responseSource: StadspasDiscountTransactionsResponseSource
-) {
-  if (Array.isArray(responseSource.transacties)) {
```

Comment:
```
  if (Array.isArray(responseSource?.transacties)) {
```
Zou ik graag in stand willen houden, hierna kun je de data transformeren, moch deze array er niet zijn return dan de responseSource zodat we kunnen zien wat daar mis mee is of iets anders mee kunnen doen. Response Errors kunnen ook een body hebben.

Reply:
Okay, begrepen. Klinkt inderdaad goed, om in iedergeval die body  te kunnen inzien als het fout gaat en vergelijken.